src/Mammon/Actors/CostCentreActor.cs

```diff
@@ -15,7 +15,12 @@ public async Task AddCostAsync(string resourceId, ResourceCost cost)
             
             state.ResourceCosts ??= [];
 
-            state.ResourceCosts.TryAdd(resourceId, cost);
+            if (!state.ResourceCosts.TryAdd(resourceId, cost))
+            {
+                //log this - this is either logical error or dapr retrying actor call
+                logger.LogWarning($"Resource {resourceId} already exists in cost centre {Id}");
```

Comment:
was this a todo or explaing why you're logging?

Reply:
yeah - trying to explain the warning part